packages/cli-doctor/src/tools/healthchecks/ruby.ts

```diff
@@ -0,0 +1,24 @@
+import versionRanges from '../versionRanges';
+import {doesSoftwareNeedToBeFixed} from '../checkInstallation';
+import {HealthCheckInterface} from '../../types';
+
+export default {
+  label: 'Ruby',
+  isRequired: false,
+  getDiagnostics: async ({Managers}) => ({
+    needsToBeFixed: doesSoftwareNeedToBeFixed({
+      version: Managers.RubyGems.version,
+      versionRange: versionRanges.RUBY,
+    }),
+    version: Managers.RubyGems.version,
+    versionRange: versionRanges.RUBY,
+  }),
+  runAutomaticFix: async ({loader, logManualInstallation}) => {
+    loader.fail();
+
+    logManualInstallation({
+      healthcheck: 'Ruby',
+      url: 'https://reactnative.dev/docs/environment-setup#ruby',
```

Comment:
@arushikesarwani, when I go to https://reactnative.dev/docs/environment-setup#ruby it only takes me to the correct place if I have previously opened the "React Native CLI Quickstart" tab (and all the other tabs underneath it) and the Ruby heading is visible. Otherwise it is a broken link.

Reply:
> yeah, unfortunately, that's a problem with the current website and how docusaurus handles deeplink with tabs. I think we need to change the website design to ease the usage of links with anchors like this.

Think its related to #1767